c2corg_ui/static/js/auth/auth.js

```diff
@@ -151,10 +161,19 @@ app.AuthController.prototype.successRegister_ = function(response) {
  * @private
  */
 app.AuthController.prototype.errorRegister_ = function(response) {
-  // TODO
-  alert('register error');
-  console.log('register error');
-  console.log(response);
+  // TODO: i18n
+  // FIXME: HTML is not interpreted in alert messages
+  var msg = 'Registration failed because of:';
+  var errors = response['data']['errors'],
+      len = errors.length;
+  if (len > 0) {
+    msg += '<ul>';
```

Comment:
As for now I have been able to render the HTML code in the messages using a ``trustAsHtml`` filter as in http://stackoverflow.com/a/30620547
But then the "close" button is lost :(